File: src/main/java/com/neblina/balero/domain/Property.java
/**
 * Balero CMS Project: Proyecto 100% Mexicano de código libre.
 * Página Oficial: http://www.balerocms.com
 *
 * @author      Anibal Gomez <anibalgomez@icloud.com>
 * @copyright   Copyright (C) 2015 Neblina Software. Derechos reservados.
 * @license     Licencia BSD; vea LICENSE.txt
 */

package com.neblina.balero.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Property {
    @Id
    @GeneratedValue
    private Long id;
    private String administratorEmail;
    private boolean offline;
    private boolean multiLanguage;
    private String url;
    private String mainLanguage;
    private String template;


    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getAdministratorEmail() {
        return administratorEmail;
    }

    public void setAdministratorEmail(String administratorEmail) {
        this.administratorEmail = administratorEmail;
    }

    public boolean isOffline() {
        return offline;
    }

    public void setOffline(boolean offline) {
        this.offline = offline;
    }

    public boolean isMultiLanguage() {
        return multiLanguage;
    }

    public void setMultiLanguage(boolean multiLanguage) {
        this.multiLanguage = multiLanguage;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getMainLanguage() {
        return mainLanguage;
    }

    public void setMainLanguage(String mainLanguage) {
        this.mainLanguage = mainLanguage;
    }

    public String getTemplate() {
        return template;
    }

    public void setTemplate(String template) {
        this.template = template;
    }

    @Override
    public String toString() {
        return "Setting [" +
                "id=" + this.id + "," +
                "administratorEmail=" + this.administratorEmail + "," +
                "offline=" + this.offline + "," +
                "multiLanguage" + this.multiLanguage + "," +
                "url" + this.url + "," +
                "mainLanguage" + this.mainLanguage + "," +
                "template" + this.template
                + "]";
    }

}

File: src/main/resources/static/scripts/service/property.js
/**
 * Balero CMS Project: Proyecto 100% Mexicano de código libre.
 * Página Oficial: http://www.balerocms.com
 *
 * @author      Anibal Gomez <anibalgomez@icloud.com>
 * @copyright   Copyright (C) 2015 Neblina Software. Derechos reservados.
 * @license     Licencia BSD; vea LICENSE.txt
 */

angular.module('PropertyService', ['ngResource'])
    .factory('PropertyService', function($resource) {
        return {
            getProperties: function() {
                return $resource('../admin/api/properties', {}, {
                    query: { method: 'GET', params: {}, isArray: false }
                })
            },
            postProperties: function(properties) {
                return $resource('../admin/api/properties', {}, {
                    save: {
                        method: 'POST',
                        params: {
                            offline: properties.offline,
                            multiLanguage: properties.multiLanguage
                        }
                    }
                })
            }
        };
    });